features/page-objects/RoomSettingPage.js

```diff
@@ -20,11 +21,8 @@ class RoomSettingPage extends Page {
   }
 
   async enterAccessCode(accessCode) {
-    const accessCodeInput = await this.findByCss("[id='waiting_room_access_code']");
-    accessCodeInput.sendKeys(accessCode.value);
-
-    const submitInput = await this.findByCss("[type='submit']");
-    return submitInput.click();
+    const waitingRoom = new WaitingRoom(this.driver)
+    waitingRoom.submitAccessCode(accessCode)
   }
```

Comment:
Is there a way we can find the usages of enterAccessCode and replace it with the new `WaitingRoom` class;so that we don't have the responsibility still split between these functions?